plugins/samples/check_pii/plugin.rs

```diff
@@ -0,0 +1,169 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START serviceextensions_plugin_check_pii]
+use proxy_wasm::traits::*;
+use proxy_wasm::types::*;
+use regex::Regex;
+use std::borrow::Cow;
+use std::sync::Arc;
+
+proxy_wasm::main! {{
+    proxy_wasm::set_log_level(LogLevel::Trace);
+    proxy_wasm::set_root_context(|_| -> Box<dyn RootContext> {
+        Box::new(MyRootContext::new())
+    });
+}}
+
+struct MyRootContext {
+    card_matcher: Option<Arc<Regex>>,
+    code10_matcher: Option<Arc<Regex>>,
```

Comment:
Can we use Rc instead of Arc here? Since we aren't interacting with multiple threads we don't need to use Arc.